<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Guy</class>
 <widget class="QDialog" name="Guy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Projects, Pods 'n Ports</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Project</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QPushButton" name="secrets">
            <property name="text">
             <string>Show Secrets</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="projects"/>
            </item>
            <item>
             <widget class="QPushButton" name="projectsRefresh">
              <property name="maximumSize">
               <size>
                <width>23</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>R</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Pods 'n Ports</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QListWidget" name="pods"/>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="infoButton">
              <property name="text">
               <string>Info</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="terminateButton">
              <property name="text">
               <string>Force Terminate!</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="taillogButton">
              <property name="text">
               <string>Tail Log</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sshButton">
              <property name="text">
               <string>SSH</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="podRefresh">
              <property name="maximumSize">
               <size>
                <width>23</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>R</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ports</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabWidget_2">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Forwarding</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QSpinBox" name="targetPort">
                    <property name="maximum">
                     <number>999999</number>
                    </property>
                    <property name="value">
                     <number>8080</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="ports"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="portsRefresh">
                    <property name="maximumSize">
                     <size>
                      <width>23</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>R</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="connectButton">
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Forward</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <widget class="QListWidget" name="forwards"/>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="stop">
                  <property name="text">
                   <string>Stop selected</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Active ports</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QTableWidget" name="portTable">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="showGrid">
                 <bool>true</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
                <attribute name="verticalHeaderDefaultSectionSize">
                 <number>23</number>
                </attribute>
                <column>
                 <property name="text">
                  <string>Protocol</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Local Port</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Remote Address</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>PID</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Action</string>
                 </property>
                </column>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Admin</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>536</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Purge unused deployments, images and pods</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>[TBI] Resource Monitor</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Config</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="3" column="0">
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Pods</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Log length</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="logLength">
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>150</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Active Ports</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Get active ports</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="getActivePorts"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="pidInfo"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="terminateConnection"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="terminateConnection_2">
            <property name="text">
             <string>Terminate connection</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="terminateConnection_4">
            <property name="text">
             <string>PID info</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="status">
     <property name="text">
      <string>Loading:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
